UPGRADE.md

```diff
@@ -33,11 +36,17 @@ Some keys have been removed from [color palette](https://github.com/CartoDB/cart
 - disabledOpacity
 - selectedOpacity
 - focusOpacity
-- other, all removed but tooltip and divider
+- other, all removed but divider, which is moved to first level
+
+Some others have been moved or replaced because they aren't native MUI keys and are so specific to some components, these are:
+
+- charts (replaced by `theme.palette.black[%]`)
+
+`grey palette` is not used to design (and therefore not for style) components directly. We have a set of neutral colors to use in the custom `default` variant.
 
-Some others have been moved because they aren't native MUI keys and are so specific to some components, these are:
+We also have a set of `shade` colors (with transparency):
 
-- charts
-- other
+- black
+- white
 
-These sets of keys are now in: `componentsPalette`.
+Important: `primary.relatedLight` and `secondary.relatedLight` has to be replaced by `primary.background` and `secondary.background`.
```

Comment:
Should we start doing it in the code, after landing this PR? or not yet?

Reply:
I already did it here in c4r, in this PR. I left this note for other platforms like cloud-native.